hw/top_darjeeling/templates/toplevel.sv.tpl

```diff
@@ -483,6 +483,14 @@ max_intrwidth = (max(len(x.name) for x in block.interrupts)
 %>\
   % if m["param_list"] or block.alerts:
   ${m["type"]} #(
+  % if 'racl_mappings' in m:
+    .EnableRacl(1'b1),
+    .RaclErrorRsp(${"1'b1" if top['racl']['error_response'] else "1'b0"}),
+    % for if_name in m['racl_mappings'].keys():
+<% if_prefix = f"_{if_name.upper()}" if if_name else "" %>\
```

Comment:
This is used as a suffix, so please rename it if_suffix.

Reply:
For sure!